src/sql/parts/queryPlan/queryPlan.ts

```diff
@@ -45,6 +45,8 @@ export class QueryPlanView implements IPanelView {
 	}
 
 	public layout(dimension: Dimension): void {
+		this.container.style.width = dimension.width.toString() + 'px';
```

Comment:
small note, you don't actually need toString, when you use `+` with a number and a string it auto converts the number to a string.